include/gmock-win32.h

```diff
@@ -808,18 +808,31 @@ struct mock_module_##func \
 #define MOCK_MODULE_FUNC_OVERLOAD(name, ...) MOCK_MODULE_UNITE(MOCK_MODULE_OVERLOAD(name, MOCK_MODULE_NBARG(__VA_ARGS__)), (__VA_ARGS__))
 #define MOCK_MODULE_FUNC(r, m, ...) MOCK_MODULE_OVERLOAD(MOCK_MODULE_FUNC, MOCK_MODULE_NBARG(__VA_ARGS__))##(m, r(__VA_ARGS__))
 
-#define MOCK_STDCALL_FUNC(r, m, ...) MOCK_MODULE_OVERLOAD(MOCK_MODULE_FUNC, MOCK_MODULE_NBARG(__VA_ARGS__)##_STDCALL_CONV)##(m, r(__VA_ARGS__))
+#define MOCK_STDCALL_FUNC(r, m, ...) \
+	MOCK_MODULE_OVERLOAD(MOCK_MODULE_FUNC, MOCK_MODULE_NBARG(__VA_ARGS__)##_STDCALL_CONV)##(m, r(__VA_ARGS__)) \
+	_Pragma( "optimize( \"\", off )" ) \
+	static void patchModuleFunc_##m() { \
+		::patchModuleFunc_( mock_module_##m::oldFn_, &::m, mock_module_##m::stub ); \
+	} \
+	_Pragma( "optimize( \"\", on )" )
+
+// Hidden from optimizer
+template <typename TFunc, typename TStub>
+static void patchModuleFunc_(void* mock_module_func_oldFn, TFunc func, TStub stub) { 
```

Comment:
Can we declare the `patchModuleFunc_` without using the `static` keyword, considering it is a template?

Reply:
At the moment, yes, statics are useless here.
I'm deleting.
Then, perhaps, it will return, there will be new improvements.
Everything should be gradual, without haste.